test/health_checker_unittest.cc

```diff
@@ -24,9 +24,13 @@ class HealthCheckerUnittest : public ::testing::Test {
 
 };
 
+typedef HealthCheckerUnittest HealthCheckerDeathTest;
+
 namespace {
-std::istringstream IsolationPathConfig(const std::string& test_name) {
-  return std::istringstream("HealthCheckFile: './" + test_name + "/unhealthy'");
+std::istringstream IsolationPathConfig(const std::string& test_name,
+                                       const std::string& extra="") {
```

Comment:
The one benefit of `IsolationPathConfig` is its uniformity. Now the reader has to wonder what the semantics of that extra text are, etc. At this point, I would forget about it and just inline the `std::istringstream` construction directly into the tests.